tests/sentry/api/endpoints/test_release_file_details.py

```diff
@@ -44,6 +44,39 @@ def test_simple(self):
         assert response.status_code == 200, response.content
         assert response.data['id'] == six.text_type(releasefile.id)
 
+    def test_file_download(self):
+        self.login_as(user=self.user)
+
+        project = self.create_project(name='foo')
+
+        release = Release.objects.create(
+            project=project,
+            organization_id=project.organization_id,
+            version='1',
+        )
+        release.add_project(project)
+
+        releasefile = ReleaseFile.objects.create(
+            organization_id=project.organization_id,
+            project=project,
+            release=release,
+            file=File.objects.create(
```

Comment:
I think you need to create the file before the release file and then use `putfile` on it:

```python
from io import BytesIO
f = File.objects.create(...)
f.putfile(BytesIO('File contents here'))
ReleaseFIle.objects.create(..., file=f)
```